packages/core/src/validate.js

```diff
@@ -252,10 +255,10 @@ export default function validateFormData(
   const errorHandler = customValidate(formData, createErrorHandler(formData));
   const userErrorSchema = unwrapErrorHandler(errorHandler);
   const newErrorSchema = mergeObjects(errorSchema, userErrorSchema, true);
-  // XXX: The errors list produced is not fully compliant with the format
-  // exposed by the jsonschema lib, which contains full field paths and other
-  // properties.
-  const newErrors = toErrorList(newErrorSchema);
+
+  // Append the user's errors to the current error list
+  //  Mantain the original error list to keep its AJV provided detail properties.
```

Comment:
Should we change typings as well because of this change?

Reply:
It would be worth exposing the "property" property on the error object.  I do not know where the typings are stored though, and could not find them in the project.

Reply:
Sorry, I was mistaken earlier -- the typescript types already include the `property` property (https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/core/index.d.ts).